app/views/people/_search.html.haml

```diff
@@ -0,0 +1,3 @@
+= form_with do |f|
+  %section{"data-controller"=>"dropdown"}
+    = f.collection_select :person_id, Person.all.sort_by(&:name), :id, :name, {:selected => person.id} , {class: "form-select w-100", "data-action": "change->dropdown#handleChange", "data-value": "/people/"}
```

Comment:
Handle case when there are no people in the database please :) 
(safe operator & olé)

Reply:
Person.all.sort_by(&:name) already returns an empty array (instead of an exception) if no people found, do you mean smth other?